### PR TITLE
Format file and thumbnails url on controller. Replace issue #123

### PR DESCRIPTION
**Pull request by matiasdelellis**
Hi @feedthedogs
Again, thank you very much for your work ... but you can try this pull-request?

### CODE REVIEW

**Comment by matiasdelellis** [comment]
@stalker314314 
The last commit is for you .. :wink:

File: js/personal.js [hunk 1]
```diff
@@ -76,23 +96,37 @@ View.prototype = {
...
         observer.observe();
 
         var self = this;
+
+        $('#facerecognition .person-name').click(function () {
+            var id = $(this).parent().data('id');
+            self._persons.loadPerson(id).done(function () {
+                self.renderContent();
+            }).fail(function () {
+                alert('D\'Oh!. Could not load person..');
```
Comment (stalker314314) [inline js/personal.js]: three dots maybe? or even unicode "…"

Comment (stalker314314) [inline js/personal.js]: is this alert, like those JS alert() functions pop-up message boxes?:)

**Comment by stalker314314** [comment]
> The last commit is for you .. wink

Oh, thank you, I am honored:stuck_out_tongue::smile:

File: js/personal.js (hunk 1 above)
Comment (matiasdelellis) [reply js/personal.js]: I have to replace them with notifications and better messages ... Although theoretically it should never be executed ..

**Pull request merged**